sds_api.py

```diff
@@ -0,0 +1,173 @@
+import datetime
+import json
+import os
+
+import requests
+
+# THESE MUST BE SET EVERY TIME FOR NOW UNTIL WE GET A DOMAIN

```

Comment:
It might be helpful to add some comments or example. Looks like at the moment, we only need to set `API_URL`. Other components are set only if set `login=True`.